charms/kfp-persistence/requirements.in

```diff
@@ -2,5 +2,8 @@
 # and https://github.com/python-jsonschema/jsonschema/issues/1114
 jsonschema<4.18
 ops
-oci-image==1.0.0
+# Use chisme from specific branch during development
+# charmed-kubeflow-chisme
+charmed-kubeflow-chisme @ git+https://github.com/canonical/charmed-kubeflow-chisme.git@add-base-charm
 serialized-data-interface<0.7
+dict
```

Comment:
Where does this get used?  I did a quick look but couldn't find it.

Reply:
Removed.